TROUBLESHOOTING.md

```diff
@@ -39,3 +39,35 @@ module.exports = {
 ```
 
 - Check github folder names if its identical to you local folder names. Sometimes github never updates your folder names even if you rename it locally. If this happens rename your folders via github or use this command `git mv <source> <destination>` and commit changes.
+
+### PROBLEM
+
+SyntaxError: Cannot use import statement outside a module
+
+### SOLUTION
+One of the node modules that is used doesn't need to be transpiled.
+
+There is a good chance that the error message shows which one:
+
+```shell
+    SyntaxError: Cannot use import statement outside a module
+      20 | <script lang="ts">
+      21 | import Vue from "vue";
+    > 22 | import Component from "../../node_modules/some-module/lib";
+         | ^
+```
+In this case **some-module** is the problem.
+
+This module needs to be ignored in the transformation process. Add the following to the
```

Comment:
Actually `transformIgnorePatterns` means: 
```
I don't want to transform x,y,z modules. 
```
However, the error `SyntaxError: Cannot use import statement outside a module` has a different meaning. 

It means
```
This module is not in the syntax which Jest can execute. You need to ask Jest to transform it.
```

The regex in your example has a `!` which tells Jest to **NOT** ignore those modules in transformation step, which means Jest needs to transform those modules.

Reply:
Thanks for pointing this out and your explanation. I will change it.